source/slang/slang-compiler.h

```diff
@@ -556,6 +556,8 @@ namespace Slang
             ///
         Scope* _getOrCreateScopeForLegacyLookup(ASTBuilder* astBuilder);
 
+        // TODO: What is proper way to get linked IR after emitPrecompiledDXIL?
+        RefPtr<IRModule> linkedIRModule;
```

Comment:
Pretty sure I should be able to get this from the output artifact, but didn't understand how.